lib/english_members_page.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class EnglishMembersPage < Scraped::HTML
+  field :member_urls do
+    noko.xpath('//table[@id="members"]/tbody/tr/td/b/a/@href').map do |row|
+      URI.join(url, row.text).to_s
```

Comment:
The built-in AbsoluteUrls decorator will do this for you: I've pushed a commit showing that.